conf/cassandra.yaml

```diff
@@ -955,14 +957,14 @@ sstable_preemptive_open_interval_in_mb: 50
 # mostly sequential IO when streaming data during bootstrap or repair, which
 # can lead to saturating the network connection and degrading rpc performance.
 # When unset, the default is 200 Mbps or 25 MB/s.
-# stream_throughput_outbound_megabits_per_sec: 200
+# stream_throughput_outbound: 200MiB/s
```

Comment:
nit/reminder: 200 Mbps != 200 MiB/s

Reply:
It is also not 25, but 24 if we talk about precision. Wondering how much to set it.

Reply:
Internally I opted in to make precise conversion. But for the default value in the cassandra.yaml I leave it 25 as 24 sounds weird